### PR TITLE
Chore  update translation for japanese to fix extra period

### PR DESCRIPTION
**Pull request by gongte1937**
#### Description of changes

Remove duplicate period for Japanese translations

#### Issue #, if available

The Japanese translation for `It may take a minute to arrive` and `Your code is on the way. To log in, enter the code we sent you` have an extra period `。`, which cause duplicate period in the message

#### Checklist

- [x] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [x] PR description included
- [x] `yarn test` passes and tests are updated/added
- [x] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax
- [x] _If this change should result in a version bump_, [changeset added](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) (This can be done after creating the PR.) This does not apply to changes made to `docs`, `e2e`, `examples`, or other private packages.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .changeset/lazy-ducks-carry.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/ui': major
```
Comment (calebpollman) [inline .changeset/lazy-ducks-carry.md:2]: This should be a minor bump:
```suggestion
'@aws-amplify/ui': minor
```

**Review by calebpollman — COMMENTED** [review]
Thanks for your contribution @gongte1937. Can you please address the comment on the changeset? Other than that looks good!

**Comment by gongte1937** [comment]
@calebpollman can you review my PR when you get a chance, thanks!

**Review by reesscot — APPROVED** [review]

**Review by calebpollman — APPROVED** [review]

**Comment by calebpollman** [comment]
> @calebpollman can you review my PR when you get a chance, thanks!

Done! Thanks again @gongte1937

**Pull request merged**